config/_default/hugo.toml

```
@@ -49,8 +49,10 @@ disableKinds = ["taxonomy", "term"]
             unsafe = true
     [markup.highlight]
         # See a complete list of available styles at https://xyproto.github.io/splash/docs/all.html
-        style = "tango"
-        lineNos = true
+        # See config options at https://gohugo.io/getting-started/configuration-markup/#highlight
+        style = "borland" # DB - Changed default style for accessibility (see https://www.ctmartin.me/blog/2019/11/print-accessible-code/)
+        lineNos = false # DB - Turned off line numbers
```

This seems like a good change, but we should discuss it and make sure that we identify where line numbers are needed.

Didn't realize that we had the option 😀

---

Can we also put something in the comment here to warn the next person that this has an impact on the copy button?
```suggestion
        lineNos = false # DB - Turned off line numbers as the default. Can turn them on individually, if required, but this causes the copy button to disappear.
```